### ISSUE TITLE
Clarify screen brightness

### ISSUE THREAD

**Issue by ace-dent** [TinyCircuits/Thumby-Website#5]
From Discord, there was some discussion if `thumby.display.brightness(0)` should turn the screen on/off.
This is based on the API docs [here](https://thumby.us/API/Graphics/#screen-brightness).

On my devices, `0` is very dim but not off. The brightness byte value  (Low `0`, Mid `28`, Hi `127` ) is passed to `SSD1306.contrast()` which sets the OLED's contrast (with CMD `0x81`) [code](https://github.com/TinyCircuits/TinyCircuits-Thumby-Code-Editor/blob/129fb940503ffedd9eabc03ca181d0618108b929/ThumbyGames/lib/ssd1306.py#L31). According to the [datasheet](https://cdn-shop.adafruit.com/datasheets/SSD1306.pdf) the resulting contrast depends on the value *and* some scale factor and an internal current reference 'Iref'. The Iref is going to be HW and implementation specific, probably varies per unit and battery charge level. Screen brightness is also affected by another value - the OLED pre-charge (CMD `0xD9`). Work has been done on this before in the Arduboy community. e.g. [1](https://community.arduboy.com/t/screen-brightness/2662). 

In conclusion, I think docs should be clear if they expect `0` to be screen off - or just the dimmest useable brightness level. The emulator should match the physical HW.

**Comment by Lennevia** [comment]
Clarified!: https://github.com/TinyCircuits/Thumby-Website/commit/0ee3cdd92a1c1573263f2b4f0d5adb4fe05e8e7f
